apis/reaper/v1alpha1/reaper_types.go

```diff
@@ -172,6 +172,10 @@ type ReaperClusterTemplate struct {
 	// and "password".
 	// +optional
 	JmxUserSecretRef corev1.LocalObjectReference `json:"jmxUserSecretRef,omitempty"`
+
+	// Defines the secret which contains the shiro.ini file controlling the Reaper UI and REST API authentication.
+	// +optional
+	ReaperUiSecretRef corev1.LocalObjectReference `json:"reaperUiSecretRef,omitempty"`
```

Comment:
For consistency I suggest renaming this field to `UiUserSecretRef`

Reply:
👍